docs-src/concepts.md

```diff
@@ -106,7 +106,12 @@ configuration. For example, two application teams may inadvertently submit
 configuration for the same HTTP path. There are several different strategies for
 handling this:
 
-* TODO
+* A controller may detect this situation during reconciliation and choose to honor
+  one resource or other, perhaps based on a timestamp (e.g. first write wins).
+  Resource `status` fields would reflect the result.
+* A [validating admission webhook](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/)
```

Comment:
Is it possible for a webhook to know state globally? I think if you have replicated API servers, you won't see all requests at all webhooks (imagine having a webhook side by side on `localhost` to the API server).

Reply:
(Maybe this is just  a caveat)